api/environment.go

```diff
@@ -57,7 +57,23 @@ func (e EnvironmentController) DeployEnvironment(c *gin.Context) {
 		return
 	}
 
-	env := h.GetEnvironment(branch, conf.R10k.Prefix, conf.R10k.AllowUppercase)
+	prefix := ""
+	switch conf.R10k.Prefix {
+	case "mapping":
```

Comment:
This appears to be a mispelling, if I'm reading it correctly, this should be looking for `mappings`, not `mapping`. 

Also, the repo uses tabs instead of spaces to conform with the standard Golang practices. If you could fix that, it'd be great, otherwise I can format things later.